queries/php/highlights.scm

```diff
@@ -8,7 +8,8 @@
  (primitive_type)
  (cast_type)
  ] @type.builtin
-(type_name (name) @type)
+(named_type (name) @type) @type
+(named_type (qualified_name) @type) @type
```

Comment:
Think the capture outside already include the inner nodes

```suggestion
(named_type (name)) @type
(named_type (qualified_name)) @type
```

or maybe you meant to capture only the inner node.